railties/lib/rails/commands/app/update_command.rb

```diff
@@ -31,6 +32,12 @@ def bin
           app_generator.update_bin_files
         end
 
+        desc "public_directory", "Add or update files in the application public/ directory", hide: true
+        def public_directory
+          require_application!
+          app_generator.create_public_files
```

Comment:
Since public files might be customized, we could instead add a `public_directory_when_updating` similar to `config_when_updating` that only creates missing files.